Jenkinsfile

```diff
@@ -166,6 +166,8 @@ def RunIntegrationTest(k, v) {
             }
             finally{
                 archive '_output/tests/*.log'
+                sh 'sudo rm -rf ${PWD}/rook-test'
```

Comment:
did you say this was no longer needed?

Reply:
still trying :)

Reply:
will keep it there till i figure out what is the best fix (likely in next PR)